docs/standard/native-interop/interop-exceptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Platform Neutral Interop Exceptions"
+ms.date: "01/16/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "unmanaged code, exceptions"
+  - "exceptions, unmanaged code"
+  - "interop, exceptions"
+  - "exceptions, interop"
+  - "stack, jump"
+  - "jump, stack"
+description: Information the use of unmanaged code interop exceptions.
```

Comment:
The other files in this directory have either description that starts with "Learn how to ..." or do not have description at all. Match one of these? I think it would be fine to not have the description for this doc - the title describes it well enough.